### ISSUE TITLE
[Accessibility] Redundant pixel show beside the bottom link of ‘Migrate NuGet format to PackageReference’ dialog

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#7151]
## Details about Problem
NuGet version: NuGet Client release-4.8.0-RTM-4.8.0.5362
VS Version: D15.8 27924.00
OS Version: Windows 10 RS4_Release 17134.1.180410-1804
## Detailed repro steps 
1. Launch VS and click menu Tools->NuGet Package Manager->Package Manager Settings.
2. Check ‘Allow format selection on first package install’ format.
3. Create a new C# Console App (.NET Framework) via VS and click ‘Manage NuGet Packages…’
4. Select a package in ‘Browse’ tab and click Install button.
5. Select ‘Packages.config’ format and click OK button.
6. Right-Click the “References” node and select the “Migrate packages.config to PackageReference…”
7. Check the bottom link on ‘Migrate NuGet format to PackageReference’ dialog.
## Expected
No redundant pixel show beside the bottom link ‘Help me migrate to NuGet PackageReference’.
## Actual
A redundant pixel show beside the bottom link ‘Help me migrate to NuGet PackageReference’ as below screenshot. 
![redundantpixel](https://user-images.githubusercontent.com/20632625/43312608-5b9be340-917d-11e8-98cb-c44aa29d9b69.png)
## Note
1.It repro on all languages.
2.The issue repro on D15.8 27924.00 (Default NuGet Client Release-4.8.0-RTM\4.8.0.5348).

**Comment by zhili1208** [comment]
@rrelyea does this should be fixed in 15.8?

**Comment by donnie-msft** [comment]
I had already found this pixel and believe it may already be fixed, but going to link to my Epic to repro later and verify.

**Comment by v-luzh** [comment]
It is fixed on Main\33912.166 for all the themes. Close this bug now. Thanks.
